dss/api/files.py

```diff
@@ -3,6 +3,8 @@
 import json
 import re
 import typing
+
+from chainedawslambda import aws
```

Comment:
The name of this import is not very specific, I'm just concerned it might be a bit confusing.

Reply:
What about just importing chainedawslambda.aws and use the full namespace?

Reply:
👍